src/patternfly/components/Popover/popover.scss

```diff
@@ -34,8 +34,7 @@
   --pf-c-popover__footer--MarginTop: var(--pf-global--spacer--lg);
 
   position: relative;
-  min-width: var(--pf-c-popover--MinWidth);
-  max-width: var(--pf-c-popover--MaxWidth);
+  display: inline-block;
```

Comment:
this shouldn't be necessary. The popover will always be in an element with `position: absolute` in react, so it will only be as big as is needed to fit the content inside up to the specified `max-width` (if that's specified)